### ISSUE TITLE
Purged items leave "ghost" database records that accumulate and break the workflow-archiver

### ISSUE THREAD

**Issue by blalbrit** [sul-dlss/workflow-archiver#23]
Summary: In its current state, the workflow-archiver cron job runs regularly and sweeps closed workflows from the workflow table to the workflow-archive table. However, the "purge" functionality in Argo (https://github.com/sul-dlss/argo/blob/f60a721af7a5007edf1c97ac359cbce39235caf0/app/controllers/items_controller.rb#L523) doesn't clean up the workflow rows for purged items - leaving ghost rows which then error repeatedly when the workflow-archiver runs.

This eventually leads to the archiver not being able to clear a backlog, and we wind up with more and more items which, for the user, become uneditable for longer than we would like, from a service perspective.

Currently, I think manually clear this list of ghost rows, but it really shouldn't require human intervention. Adding a ticket to the Argo GH project as well, and proposing to @vivnwong for a maintenance fix.

**Comment by blalbrit** [comment]
Oh hey - looks like I filed this same ticket in October of 2017 in the Argo project: https://github.com/sul-dlss/argo/issues/1020

**Comment by aaron-collier** [comment]
Tracking: https://jirasul.stanford.edu/jira/browse/DEVQUEUE-231

**Comment by mjgiarlo** [comment]
Blocked by requiring a team discussion, planned for Tuesday, 12/11/2018.

**Comment by peetucket** [comment]
related to sul-dlss/argo#1020

**Comment by mjgiarlo** [comment]
Duplicate of https://github.com/sul-dlss/argo/issues/1020, and work will be done in Argo. Closing in workflow-archiver.
